com.vrcfury.vrcfury/Editor/VF/Feature/Base/FeatureBuilder.cs

```diff
@@ -148,6 +148,25 @@ public abstract class FeatureBuilder {
                         }
                         break;
                     }
+                    case PoiyomiUVTileAction poiyomiUVTileAction: {
+                        var renderer = poiyomiUVTileAction.renderer;
+                        if (poiyomiUVTileAction.slot1 > 3 || poiyomiUVTileAction.slot1 < 0) {
+                            Debug.LogWarning("Poiyomi UV tile Discard only has 4 slots in the UV X axis.");

```

Comment:
We should probably just throw an exception in these, and then the checks below on 163 aren't needed. Better to be strict early than want to be strict later. If we allow invalid inputs, it means we have to continue supporting them forever.

Reply:
I do agree, It was a simple addition of a warning and if anyone wants to improve on it I don't have any issues.

Reply:
I added a throw exemption. With a simple explanation that the values for uv tile discard are 0-3.